examples/apache/build-apache.acb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env acbuild-script
+
+# Start the build with an empty ACI
+begin
+
+# Name the ACI
+set-name example.com/apache
+
+# Based on alpine
+dep add quay.io/coreos/alpine-sh
+
+# Install apache
+run -- apk update
```

Comment:
doesn't `--` need to be removed (since you updated it to assume it when running `run` options, etc?)